### ISSUE TITLE
没有没有响应，持续发发送没有响应。

### ISSUE THREAD

**Issue by animeshawtylovemealot** [dice2o/BingGPT#161]
没有反映呢，还可以持续发文。

![Snipaste_2023-04-25_15-43-06](https://user-images.githubusercontent.com/80526156/234208598-c138316a-907e-46a9-bcb5-f0c1e8c66bef.png)

**Comment by iRubbish** [comment]
我也是，撰写正常对话框 就是没反应

**Comment by skywk** [comment]
同样的问题，聊天输入无响应。但是撰写正常.

**Comment by chenyusheng** [comment]
me too

**Comment by ErrorPower2001** [comment]
一样，也就一个小时前

**Comment by galanz01** [comment]
me too

**Comment by Mr-gan8** [comment]
me too

**Comment by Mr-gan8** [comment]
How to solve it?

**Comment by fwilyair** [comment]
一样，网页测试确认账号和梯子没有问题，撰写功能正常，就是输入无反应

**Comment by Liaix** [comment]
me too

**Comment by huachuman** [comment]
Yes, it seems there must have been a recent change that made this edgeservices URL not work anymore.

**Comment by huachuman** [comment]
Here's a possible lead: https://chrome.google.com/webstore/detail/bing-chat-for-all-browser/jofbglonpbndadajbafmmaklbfbkggpo

This chrome extension loads a URL that is currently working.
